### ISSUE TITLE
Has_year property needed and unavailable for most -> too many events

### ISSUE THREAD

**Issue by WolfgangFahl** [SmartDataAnalytics/OpenResearch#197]
https://confident.dbis.rwth-aachen.de/or/index.php/Property:Has_year

**Comment by WolfgangFahl** [comment]
```bash
wikipush  -l -s rq -t orclone -p "Variables"
copying 1 pages from rq to orclone
1/1 ( 100%): copying Variables ...✅
```

**Comment by WolfgangFahl** [comment]
https://www.mediawiki.org/wiki/Special:ExtensionDistributor?extdistname=Variables&extdistversion=REL1_31

**Comment by WolfgangFahl** [comment]
```bash
wf@confident:/var/www/mediawiki/or/extensions$ tar xvfz /usr/local/src/Variables-REL1_31-bb5f5b2.tar.gz
chgrp -R www-data Variables/
```

**Comment by WolfgangFahl** [comment]
needs #83 to be fixed due to MediaWiki 2.5.8 not supporting the needed functionality
